unbxdSearch.js

```diff
@@ -1277,16 +1372,36 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
 
             return this;
         },
-        setDefaultParams: function(params) {
+        setDefaultParams: function (params) {
             var newparams, diff;
             var oldparams = JSON.stringify(params);
             this.options.setDefaultFilters.call(this);
+
+            this.params['extra']['version'] = 'V2';
+
+            if (this.options.facetMultilevel) {
+                this.params['extra']['facet.multilevel'] = 'categoryPath';
+                this.params['extra']['f.categoryPath.displayName'] = this.options.facetMultilevelName;
+                this.params['extra']['f.categoryPath.max.depth'] = '6';
+                this.params['extra']['f.categoryPath.facet.limit'] = '100';
```

Comment:
Not needed to be `100`, it can be limited based on the customer.

Reply:
Done